contents/handbook/engineering/support-hero.md

```diff
@@ -4,35 +4,59 @@ sidebar: Handbook
 showTitle: true
 ---
 
-## 1. Support Hero
+Every week, one person in each engineering team is designated the Support Hero. If this is you this week, congratulations!
 
-Every week, we assign one person to be the "Support Hero" per engineering team (alternative names "Secondary", "Support Sidekick", "Infra Hero", "Luigi"). If this is you this week, congratulations! Support hero is an intense but super fun week where you get to talk to a bunch of users, get the satisfaction of helping them out, and contribute to a lot of different parts of our system. Your first priority should be dealing with alerts or Sentry alerts are high priority. After that, it's responding to customer support requests. Depending on how busy the week is you can do some feature work too.
+As Support Hero, your job is to investigate and resolve issues reported by customers. A single case of suspicious data or a show-stopping bug can really undermine one's confidence in a data product, so it's important that we get to the bottom of all issues.
 
-[Marcus](https://posthog.com/community/profiles/1036), our Support Engineer, will triage tickets for the Product Analytics and Pipeline team, due to the high volume of tickets those teams get. He will resolve tickets if possible, and escalate to the Engineering team responsible if he needs further help.
+You'll see some teams using a term of endearment for Support Hero, examples being "Infra Hero" or… "Luigi". Don't ask – we don't know.
 
-### Expectations
+[Marcus](https://posthog.com/community/profiles/1036), our Support Engineer, triages tickets for the Product Analytics and Pipeline team, due to the high volume of tickets those teams get. He will resolve tickets if possible, and escalate to the engineering team responsible if he needs further help.
 
-You should work your 'normal' hours during this week. There will likely be more issues than you'll have time to fix so be sure to prioritise. If there's an important and urgent issue near the end of your day, hand it off to someone on the other side of the Atlantic.
+## When is my turn?
 
-If you are planning on taking a day off or you won't be available, please find someone to swap with and update the rotation on PagerDuty. Be sure to schedule an override for both swaps and **do not alter the rotation order** to avoid affecting other members.
+Most engineering teams run a PagerDuty schedule:
 
-### Rotations
+- [Product Analytics](https://posthog.pagerduty.com/schedules#PXUZ9XL)
+- [Feature Success](https://posthog.pagerduty.com/schedules#P04FUTJ)
+- [Replay](https://posthog.pagerduty.com/schedules#PUPERAV)
```

Comment:
This link is broken, maybe https://posthog.pagerduty.com/schedules#PLGXQIF is the new one?

Reply:
Oh yeah
```suggestion
- [Replay](https://posthog.pagerduty.com/schedules#PLGXQIF)
```